packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -188,13 +192,14 @@ export const SELL_SOURCE_FILTER_BY_CHAIN_ID = valueByChainId<SourceFilters>(
             ERC20BridgeSource.SpiritSwap,
             ERC20BridgeSource.SpookySwap,
             ERC20BridgeSource.SushiSwap,
+            ERC20BridgeSource.Synapse,
         ]),
         [ChainId.Celo]: new SourceFilters([
             ERC20BridgeSource.UbeSwap,
             ERC20BridgeSource.SushiSwap,
             ERC20BridgeSource.MultiHop,
         ]),
-        [ChainId.Optimism]: new SourceFilters([ERC20BridgeSource.UniswapV3]),
+        [ChainId.Optimism]: new SourceFilters([ERC20BridgeSource.UniswapV3, ERC20BridgeSource.Synapse]),
```

Comment:
Can you also drop in `ERC20BridgeSource.MultiHop` here
```suggestion
        [ChainId.Optimism]: new SourceFilters([
            ERC20BridgeSource.UniswapV3,
            ERC20BridgeSource.Synapse,
            ERC20BridgeSource.MultiHop,
         ]),
```